integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java

```diff
@@ -838,6 +866,162 @@ public void testErrorPathFilePermission() {
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    String encryptionSecretName = "encryptionsecret";
+    createSecretWithUsernamePassword(encryptionSecretName, wdtDomainNamespace,
+        "weblogicenc", "weblogicenc");
+
+    // create stage dir for first auxiliary image containing domain configuration
+    Path multipleAIPath1 = Paths.get(RESULTS_ROOT, "multipleauxiliaryimage1");
+    assertDoesNotThrow(() -> FileUtils.deleteDirectory(multipleAIPath1.toFile()),
+        "Delete directory failed");
+    assertDoesNotThrow(() -> Files.createDirectories(multipleAIPath1),
+        "Failed to create directory");
+
+    // create models dir and copy model, archive files if any for image1
+    Path modelsPath1 = Paths.get(multipleAIPath1.toString(), "models");
+    assertDoesNotThrow(() -> Files.createDirectories(modelsPath1));
+    assertDoesNotThrow(() -> Files.copy(
+        Paths.get(MODEL_DIR, MII_BASIC_WDT_MODEL_FILE),
+        Paths.get(modelsPath1.toString(), MII_BASIC_WDT_MODEL_FILE),
+        StandardCopyOption.REPLACE_EXISTING), "Failed to copy file");
+    assertDoesNotThrow(() -> Files.copy(
+        Paths.get(MODEL_DIR, "multi-model-one-ds.20.yaml"),
+        Paths.get(modelsPath1.toString(), "multi-model-one-ds.20.yaml"),
+        StandardCopyOption.REPLACE_EXISTING), "Failed to copy file");
+
+    // build app
+    assertTrue(buildAppArchive(defaultAppParams()
+            .srcDirList(Collections.singletonList(MII_BASIC_APP_NAME))
+            .appName(MII_BASIC_APP_NAME)),
+        String.format("Failed to create app archive for %s", MII_BASIC_APP_NAME));
+
+    // copy app archive to models
+    assertDoesNotThrow(() -> Files.copy(
+        Paths.get(ARCHIVE_DIR,  MII_BASIC_APP_NAME + ".zip"),
+        Paths.get(modelsPath1.toString(), MII_BASIC_APP_NAME + ".zip"),
+        StandardCopyOption.REPLACE_EXISTING), "Failed to copy file");
+
+    // create image1 with domain configuration only
+    createAuxiliaryImage(multipleAIPath1.toString(),
+        Paths.get(RESOURCE_DIR, "auxiliaryimage", "Dockerfile").toString(), miiAuxiliaryImage4);
+
+    // push image1 to repo for multi node cluster
+    if (!DOMAIN_IMAGES_REPO.isEmpty()) {
+      logger.info("docker push image {0} to registry {1}", miiAuxiliaryImage4, DOMAIN_IMAGES_REPO);
+      dockerLoginAndPushImageToRegistry(miiAuxiliaryImage4);
+    }
+
+    // create stage dir for second auxiliary image
+    Path multipleAIPath2 = Paths.get(RESULTS_ROOT, "multipleauxiliaryimage2");
+    assertDoesNotThrow(() -> FileUtils.deleteDirectory(multipleAIPath2.toFile()),
+        "Delete directory failed");
+    assertDoesNotThrow(() -> Files.createDirectories(multipleAIPath2), "Create directory failed");
+
+    Path modelsPath2 = Paths.get(multipleAIPath2.toString(), "models");
+    assertDoesNotThrow(() -> Files.createDirectories(modelsPath2), "Create directory failed");
+
+    // unzip 1.9.15 version WDT installation file into work dir
```

Comment:
Can you make the WDT version as a variable in TestConstant class instead of hard-coding to v 1.9.5

Reply:
replaced with variable WDT_TEST_VERSION, it could be set in jenkin for different value or used default (1.9.15 previous from latest version)